### PR TITLE
Rename docker denoms

### LINKED ISSUE #57: Rename denom to ucosm and ustake
The [test chain from docker](https://hub.docker.com/r/cosmwasm/wasmd) comes with two tokens, COSM and STAKE. Both have 6 fractional digits. Thus the `denom` value should be µCOSM and µSTAKE, or "ucosm" and "ustake" for consistency with ATOMs.

This needs to be updated in client-side configuration once it is changed in the blockchain config.

### PR DESCRIPTION
**Pull request by ethanfrey**
Closes #57 

Makes `ustake` the staking/governance token. `ucosm` is a second fee token

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docker/setup.sh [hunk 1]
```diff
@@ -3,13 +3,15 @@
 PASSWORD=${PASSWORD:-1234567890}
 
 wasmd init --chain-id=testing testing
+# staking/governance token is hardcoded in config, change this
+sed -i 's/"stake"/"ustake"/' "$HOME"/.wasmd/config/genesis.json
```
Comment (webmaster128) [inline docker/setup.sh]: no similar diff for cosm -> ucosm? Is the token created by the first account receiving it?

Comment (ethanfrey) [reply docker/setup.sh]: The account balancea are defines un the creaste account command below.

The default genesis hard codes stake as token for staking validators and voting in governance. Sed fixes that (no cli flag)

Comment (ethanfrey) [reply docker/setup.sh]: Actually, maybe we make both denoms args to this script, so we control in the package where we run in CI, not here. And then default to these denoms if $1 and $2 are undefined

Comment (webmaster128) [reply docker/setup.sh]: But where is the COSM token configuted? At which point does the token come into existence?

Comment (ethanfrey) [reply docker/setup.sh]: When you add it to a genesis account.

There is no explicit definition of tokens or minting.
They look at balances in genesis and those are the tokens on the chain.

I know, you miss weave :)

Comment (webmaster128) [reply docker/setup.sh]: Okay cool, thanks, that answers my question.

And it answers much more than that. In order to get a list of chain tokens, you need to iterate over all accounts on the chain 🤯

Comment (ethanfrey) [reply docker/setup.sh]: They added a supply module recently that basically does that iteration. I should give a list of denoms, but no fractional digits or other info

**Pull request merged**